fixtures/BasketFixture.php

```diff
@@ -248,6 +248,28 @@ public function nine()
     }
 
     /**
+     * Products: 2
+     * Taxable:  2
+     * Value:    £1009.99
+     * Delivery: £0
+     * Discount: £202
+     * Tax:      £182
+     * Subtotal: £909.99
+     * Total:    £1091.99
+     *
```

Comment:
Subtotal = ? Value - Discount?
Total = Subtotal + Tax

Nem értem a példa számokat, nem világos, hogy hogyan jön ki Subtotal. Persze ez itt nem érdekes, majd a dokumentációban esetleg pontosítsd, hogy jobban érthető legyen a példa. Elméletileg ahogyan működnie kellene az így lenne:

Feltételezem, hogy a kiszállítási szolgáltatásra általában 0%-os adót kell számolni, de jobb ha felkészülünk a meglepikre.

// Termék ÁFA
TaxRate = tt%
// Kiszállítási szolgáltatási díj adókulcsa
DeliveryTaxRate = dd%
// Az eladott termékre vonatkozó adó mértéke
ValueTax = Value * ( TaxRate / 100 )
// A kiszállításra vonatkozó adó mértéke
DeliveryTax = Delivery * (DeliveryTaxRate / 100)
// A kiszállítási költség nettó értéke (mivel a kiszállítás díja bruttó összegként lesz beállítható)
DeliveryNet = Delivery - DeliveryTax
// Adó nélkül bevétel összesen
Subtotal (Net Total) = Value + DeliveryNet - Discount
// Adó összesen
Tax = ValueTax + DeliveryTax

Még az a kérdés, hogy a mi részesedésünk a Total vagy a Subtotal összegre számolódik -e majd.

Bocs, hogy ide jegyzetelem ezt is, talán később hasznos lehet.